clients/src/main/resources/common/message/ConsumerProtocolSubscription.json

```diff
@@ -21,7 +21,10 @@
   // The current implementation assumes that future versions will not break compatibility. When
   // it encounters a newer version, it parses it using the current format. This basically means
   // that new versions cannot remove or reorder any of the existing fields.
-  "validVersions": "0-1",
+  //
+  // Version 1 adds owned partitions.
+  // Version 2 adds rack id to enable rack-aware assignment.
+  "validVersions": "0-2",
```

Comment:
Will update to use version 3 when https://github.com/apache/kafka/pull/12748 is merged (KIP-792).

Reply:
#12748 is merged. Thanks.

Reply:
Thanks @showuon , rebased and updated the version.